addons/website_sale/models/product.py

```diff
@@ -339,6 +342,23 @@ def _get_current_company_fallback(self, **kwargs):
         website = self.website_id or kwargs.get('website')
         return website and website.company_id or res
 
+    def _init_column(self, column_name):
+        # to avoid generating a single default website_sequence when installing the module,
+        # we need to set the default row by row for this column
+        if column_name == "website_sequence":
+            _logger.debug("Table '%s': setting default value of new column %s to unique values for each row", self._table, column_name)
+            self.env.cr.execute("SELECT id FROM %s WHERE website_sequence IS NULL" % self._table)
+            prod_tmpl_ids = self.env.cr.dictfetchall()
+            prod_tmpl_num = len(prod_tmpl_ids)
+            max_seq = self._default_website_sequence()
+            prod_string = ", ".join([str(
+                (prod_tmpl_id['id'], max_seq + (prod_tmpl_num - prod_tmpl_id['id']) * 5)
+            ) for prod_tmpl_id in prod_tmpl_ids])
+            query = 'UPDATE %s SET website_sequence = p.web_seq FROM (VALUES %s) AS p(p_id, web_seq) WHERE id = p.p_id;' % (self._table, prod_string)
+            self.env.cr.execute(query)
+        else:
+            super(ProductTemplate, self)._init_column(column_name)
```

Comment:
```suggestion
    def _init_column(self, column_name):
        # to avoid generating a single default website_sequence when installing the module,
        # we need to set the default row by row for this column
        if column_name == "website_sequence":
            _logger.debug("Table '%s': setting default value of new column %s to unique values for each row", self._table, column_name)
            self.env.cr.execute("SELECT id FROM %s WHERE website_sequence IS NULL" % self._table)
            prod_tmpl_ids = self.env.cr.dictfetchall()
            max_seq = self._default_website_sequence()
            query = sql.SQL('UPDATE {table} SET website_sequence = p.web_seq FROM (VALUES {vals}) AS p(p_id, web_seq) WHERE id = p.p_id').format(
                table=sql.Identifier(self._table),
                vals=sql.SQL(', ').join(sql.Composed([
                    sql.Composed([sql.SQL('('), sql.Literal(prod_tmpl_id['id']), sql.SQL(', '), sql.Literal(max_seq + i * 5), sql.SQL(')')])
                    for i, prod_tmpl_id in enumerate(prod_tmpl_ids)
                ]))
            )
            self.env.cr.execute(query)
        else:
            super(ProductTemplate, self)._init_column(column_name)
```

Reply:
Don't forget the import of `sql` from `psycopg2`
```py
from psycopg2 import sql
```